tinyagentos/services/rknn_sd_server.py

```
@@ -165,13 +195,30 @@ async def _ensure_pipeline():
+
+
 @app.on_event("startup")
 async def _startup():
-    # Intentionally empty: the pipeline is now lazy-loaded on the first
-    # /generate request. Old behaviour was to build the full pipeline
-    # here, which pinned ~5.5 GB of RAM at boot for users who never
-    # actually used image gen.
+    # Pipeline is lazy-loaded on the first /generate request.
+    # Old behaviour was to build here, pinning ~5.5 GB of RAM at boot.
     logger.info("rknn_sd_server up — pipeline will lazy-load on first /generate request")
+    asyncio.create_task(_idle_unload_loop())
```

Ruff (0.15.12)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)